src/main/java/com/uid2/shared/store/ACLSnapShotAccessMethod.java

```diff
@@ -0,0 +1,6 @@
+package com.uid2.shared.store;
+
+public enum ACLSnapShotAccessMethod {
```

Comment:
I got confused by this enum as ACL is used in both cases. Maybe something like:

AclMode.LegacyDep
AclMode.Sharing

?